### PR TITLE
multimc: Update to version 0.6.14

### PR DESCRIPTION
**Pull request by UnicorNora**

### CODE REVIEW

**Comment by UnicorNora** [comment]
Wget is causing this to fail

**Comment by UnicorNora** [comment]
Fixes #511

File: bucket/multimc.json [hunk 1]
```diff
@@ -34,6 +34,6 @@
         "github": "https://github.com/MultiMC/MultiMC5"
     },
     "autoupdate": {
-        "url": "https://github.com/MultiMC/MultiMC5/releases/download/$version/mmc-stable-win32.zip"
+        "url": "https://files.multimc.org/downloads/mmc-stable-win32.zip"
     }
-}
+}
```
Comment (Calinou) [inline bucket/multimc.json:39]:
```suggestion
}

```

Comment (UnicorNora) [reply bucket/multimc.json]: Whats the difference is it just a line ending?

**Pull request merged**